src/api/include/migraphx/migraphx.hpp

```diff
@@ -571,10 +571,91 @@ using require_interface =
 // NOLINTNEXTLINE
 #define MIGRAPHX_CONST_HANDLE_BASE(name) MIGRAPHX_DETAIL_HANDLE_BASE(name, const)
 
+/**
+ * Container to hold optimal dynamic dimension values.
+ */
+struct optimals : MIGRAPHX_HANDLE_BASE(optimals)
+{
+    MIGRAPHX_HANDLE_CONSTRUCTOR(optimals)
+
+    optimals(std::initializer_list<size_t> init_list)
+    {
+        std::vector<size_t> a{init_list.begin(), init_list.end()};
+        this->make_handle(&migraphx_optimals_create, a.data(), a.size());
```

Comment:
You should be able to do `this->make_handle(&migraphx_optimals_create, init_list.begin(), init_list.size())`.